### PR TITLE
Display ASCII tables' scalars indented to the right #83

### PR DESCRIPTION
**Pull request by todvora**

### CODE REVIEW

**Comment by todvora** [comment]
I found one minor issue in the significant digits count. It should be fixed now.

**Comment by JamesCropcho** [comment]
Tomáš, I have examined the pull's Variety output locally, including with irrational numbers, and everything looks good to me, at least. Thanks, man. It looks great as we all know and expect.

Do you think you could quickly update the README, too? This would eliminate confusion and show off our nice new formatting :)


James

**Comment by todvora** [comment]
The readme is updated now, thanks for the code review :-)

Tomas

**Comment by JamesCropcho** [comment]
Wonderful. Lookin' good, and is now merged!

File: README.markdown [hunk 1]
```diff
@@ -31,13 +31,13 @@ So, let's see what we've got here:
     +------------------------------------------------------------+
     | key                | types        | occurrences | percents |
     | ------------------ | ------------ | ----------- | -------- |
-    | _id                | ObjectId     | 5           | 100      |
```
Comment (Acetolyne) [inline README.markdown]: T

**Pull request merged**